packages/lib-core/src/utils/__tests__/logger.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { consoleLogger } from '../logger';
+
+describe('consoleLogger', () => {
```

Comment:
Anyway we can test for env variables? Because that'd be a good test too to make sure `info` is not doing anything in production.

Reply:
Yeah, we can do that! I'll ad it there.